src/GovUk.Education.ExploreEducationStatistics.Admin/Services/Util/PermissionsUtils.cs

```diff
@@ -1,28 +1,52 @@
+#nullable enable
 using System.Threading.Tasks;
 using GovUk.Education.ExploreEducationStatistics.Admin.Services.Interfaces.Security;
 using GovUk.Education.ExploreEducationStatistics.Admin.ViewModels;
 using GovUk.Education.ExploreEducationStatistics.Common.Model;
 using GovUk.Education.ExploreEducationStatistics.Common.Services.Interfaces.Security;
 using GovUk.Education.ExploreEducationStatistics.Content.Model;
 using Microsoft.AspNetCore.Mvc;
+using static GovUk.Education.ExploreEducationStatistics.Admin.ViewModels.MethodologyVersionSummaryViewModel;
 
 namespace GovUk.Education.ExploreEducationStatistics.Admin.Services.Util;
 
 public static class PermissionsUtils
 {
-   public static ReleasePermissions GetReleasePermissions(IUserService userService, Release release)
-   {
-       return new ReleasePermissions
-       {
-           CanAddPrereleaseUsers = CheckHasPermission(userService.CheckCanAssignPrereleaseContactsToRelease(release)),
-           CanUpdateRelease = CheckHasPermission(userService.CheckCanUpdateRelease(release)),
-           CanDeleteRelease = CheckHasPermission(userService.CheckCanDeleteRelease(release)),
-           CanMakeAmendmentOfRelease = CheckHasPermission(userService.CheckCanMakeAmendmentOfRelease(release))
-       };
-   }
+    public static ReleasePermissions GetReleasePermissions(IUserService userService, Release release)
+    {
+        return new ReleasePermissions
+        {
+            CanAddPrereleaseUsers = CheckHasPermission(userService.CheckCanAssignPrereleaseContactsToRelease(release)),
+            CanUpdateRelease = CheckHasPermission(userService.CheckCanUpdateRelease(release)),
+            CanDeleteRelease = CheckHasPermission(userService.CheckCanDeleteRelease(release)),
+            CanMakeAmendmentOfRelease = CheckHasPermission(userService.CheckCanMakeAmendmentOfRelease(release))
```

Comment:
We could make all these async using the `IsRight` method? We could get rid of the `CheckHasPermission` method as well

Reply:
The reason why I didn't do this before was because of down stream changes that it would require, which were nothing to do with my original work, but I've done this now in d03401b3b51428e652d67a14a9fb3c0c7560d2f7 🙂 .